### PR TITLE
fix(behavior_velocity): traffic light missing debug wall pose

### PR DESCRIPTION
**Pull request by taikitanaka3**
## Description

- bug fix for debug marker

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: planning/behavior_velocity_planner/src/scene_module/traffic_light/scene.cpp [hunk 1]
```diff
@@ -465,6 +465,7 @@ autoware_auto_planning_msgs::msg::PathWithLaneId TrafficLightModule::insertStopP
   target_point_with_lane_id.point.pose.position.x = target_point.x();
   target_point_with_lane_id.point.pose.position.y = target_point.y();
   target_point_with_lane_id.point.longitudinal_velocity_mps = 0.0;
+  debug_data_.stop_pose.push_back(target_point_with_lane_id.point.pose);
```
Comment (tkimura4) [inline planning/behavior_velocity_planner/src/scene_module/traffic_light/scene.cpp:468]: stop_poses, じゃないですかね

Comment (taikitanaka3) [inline planning/behavior_velocity_planner/src/scene_module/traffic_light/scene.cpp:468]:
```suggestion
  debug_data_.stop_poses.push_back(target_point_with_lane_id.point.pose);
```

Comment (taikitanaka3) [reply planning/behavior_velocity_planner/src/scene_module/traffic_light/scene.cpp]: fixed at 
[f8bee47](https://github.com/autowarefoundation/autoware.universe/pull/689/commits/f8bee4771a2f08573d128c96d40b2ba30a85ff6c)

**Review by tkimura4 — APPROVED** [review]
I comfirmed that the "wall" debug_marker is published using planning_simulator.

**Pull request merged**